dev/README.md

```diff
@@ -0,0 +1,15 @@
+# DEV 
+
+We provide a `docker-compose.yaml` file that you can use to spin up all the resources that the manager needs to run (keycloak, postgres, prometheus and grafana). 
+
+To start it, run from the current directory
+
+```bash
+docker-compose up
+```
+
+If you want to generate some traffic automatically, we provide the script `generate_traffic.py` that you can run with 
```

Comment:
As far as I see it is a `while(true)` script so please describe also the shortcut to stop it :)